OpenSim/Common/TimeSeriesTable.h

```diff
@@ -123,6 +123,35 @@ class TimeSeriesTable_ : public DataTable_<double, ETY> {
                          TimeColumnNotIncreasing);
     }
 
+    /** Construct TimeSeriesTable_ from a file.                           
+    
+    \param filename Name of the file. File should contain only one table. For
+                    example, trc, csv & sto files contain one table whereas a 
+                    c3d file can contain more than.                           
+
```

Comment:
Did you consider adding `tableName` as an optional argument to handle the case that multiple tables are returned by the adapter? Then, there would be a single interface to handle all supported file formats.

For example, it would be great to do IK with a data source that could be a: .trc, .c3d, or .sto.